code/modules/uplink/uplink_items.dm

```diff
@@ -1393,6 +1394,20 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	cost = 40
 	cant_discount = TRUE
 
+//Race-specific items
+/datum/uplink_item/race_restricted
+	category = "Species-Restricted"
+	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
+	surplus = 0
+
+/datum/uplink_item/race_restricted/syndilamp
+	name = "Extra-Bright Lantern"
+	desc = "The Syndicate heard that moths such as yourself really like lamps, \
+	so they decided you can get early access to a prototype Syndicate brand \"Extra-Bright Lantern\"."
```

Comment:
Actually now that I think about it this message is phrased wrong since it's supposed to be the Syndicate speaking, not a narrator. Instead of "The Syndicate heard" maybe just put "We heard" and instead of "they" put "we"
Also "to grant you early access" may sound better than "you can get early access"

Reply:
I'll fix it later, editing on mobile is way too awkward.